lunatrace/bsl/docker-compose.yaml

```diff
@@ -101,8 +101,8 @@ services:
       # TODO: ADD THIS TO THE CDK
       REMOTE_SCHEMA_URL: http://host.docker.internal:3002/graphql/v1
       # COMMENT OUT THESE DIRs IF HASURA WONT START DUE TO MIGRATION OR METADATA ISSUES
-      HASURA_GRAPHQL_METADATA_DIR: /hasura/metadata
-      HASURA_GRAPHQL_MIGRATIONS_DIR: /hasura/migrations
+#      HASURA_GRAPHQL_METADATA_DIR: /hasura/metadata
```

Comment:
This is just causing problems on the daily for me, I think we have to drop it @ajvpot  unless anyone feels strongly in which case ill come up with a local override.

Reply:
nevermind ill override it locally